src/components/CalendarDay.js

```diff
@@ -10,7 +10,9 @@ const CalendarDay = ({day, isToday, events}) => {
     });
 
     return (
-        <div className={dayClasses}>
+      <div 
+        onClick={onClick.bind(null, 'day', day)}
```

Comment:
Do you see a way to get to the events for that day? In `CalendarDay` they are already rendered...